src/app/containers/RadioSchedule/Canonical/index.jsx

```diff
@@ -67,7 +68,8 @@ const CanonicalRadioSchedule = ({ endpoint }) => {
     ServiceContext,
   );
   const { timeOnServer } = useContext(RequestContext);
-
+  moment.locale('en-GB');
```

Comment:
Just wondering why we're setting the moment locale to 'en-GB' here?

Reply:
I don't think we need it either. Will remove